2.5/test/run

```diff
@@ -144,30 +146,55 @@ test_scl_usage() {
     echo "ERROR[exec /bin/sh -ic "${run_cmd}"] Expected '${expected}', got '${out}'"
     return 1
   fi
+  return 0
+}
+
+function test_scl_usage_functionality() {
+  test_scl_usage "ruby --version" "ruby ${RUBY_VERSION}."
```

Comment:
I don't see how wrapper is benefitial to a default or hard-coded value?

Reply:
wrapper is used because of I do not want to add "ruby --version" into TEST_LIST variable.